### PR TITLE
*: release related improvements

### PR DESCRIPTION
**Pull request by AnnaShaleva**
In this PR:
* add CHANGELOG
* add release instructions
* take care of GitHub workflows

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changelog
+
+This document outlines major changes between releases.
```
Comment (roman-khimov) [inline CHANGELOG.md:3]: 0.1.0 should be mentioned as initial one with a link.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+## [Unreleased]
```
Comment (roman-khimov) [inline CHANGELOG.md:5]: Then "unreleased" can point to commits since 0.1.0, see neofs-node for example.

File: CHANGELOG.md (hunk 1 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:3]: > 0.1.0 should be mentioned as initial one
Fixed.

> with a link.
Where to put the link?

File: CHANGELOG.md (hunk 2 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:5]: I took this from neofs node, it looks exactly like they have. Not sure how should I change it.

Comment (AnnaShaleva) [reply CHANGELOG.md:5]: BTW, are you sure we need the "Unreleased" section at all? We don't have it in NeoGo and it works fine.

File: CHANGELOG.md (hunk 1 above)
Comment (roman-khimov) [reply CHANGELOG.md:3]: At the bottom, like
```
[Unreleased]: https://github.com/nspcc-dev/neofs-node/compare/v0.40.0...master
[0.40.0]: https://github.com/nspcc-dev/neofs-node/compare/v0.39.2...v0.40.0
[0.39.2]: https://github.com/nspcc-dev/neofs-node/compare/v0.39.1...v0.39.2
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * add generic Hash/Address parameters to `DBFT` service (#94)
+ * remove custom payloads implementation from default `DBFT` service configuration
+   (#94)
+
+Improvements:
+ * untie `dbft` module from `github.com/nspcc-dev/neo-go` dependency (#94)
+
+Bugs fixed:
+
+## 0.1.0 (13 Feb 2024)
```
Comment (roman-khimov) [inline CHANGELOG.md:19]: You can safely put 15 May 2023 here, the date of commit.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+New features:
+
+Behaviour changes:
+ * add generic Hash/Address parameters to `DBFT` service (#94)
+ * remove custom payloads implementation from default `DBFT` service configuration
+   (#94)
+
+Improvements:
+ * untie `dbft` module from `github.com/nspcc-dev/neo-go` dependency (#94)
+
```
Comment (roman-khimov) [inline CHANGELOG.md:16]: There also were dependency upgrades of various kinds.

File: CHANGELOG.md (hunk 2 above)
Comment (roman-khimov) [reply CHANGELOG.md:5]: I'm OK either way.

File: CHANGELOG.md (hunk 1 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:3]: Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:5]: OK, let's try to keep unreleased for now.

File: CHANGELOG.md (hunk 3 above)
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 4 above)
Comment (AnnaShaleva) [reply CHANGELOG.md:23]: Fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**